docs/content/ingestion/update-existing-data.md

```diff
@@ -75,8 +76,7 @@ For example
 
 #### `multi`
 
-This is a composing inputSpec to combine other inputSpecs. This inputSpec is used for delta ingestion. 
-Please note that delta ingestion is not an idempotent operation. We may add change things in future to make it idempotent.
+This is a composing inputSpec to combine other inputSpecs. This inputSpec is used for delta ingestion. Please note that you can have only one `dataSource` as child of `multi` inputSpec.
```

Comment:
I think we should document the process, it's probably too much to expect people to figure it out themselves.

1. Call the coordinator api to get the list of segments
2. Submit a "multi" task with a "dataSource" child that does include those segments
3. If you get a failure, go back to step 1 and repeat until you succeed